telegram/ext/updater.py

```diff
@@ -97,58 +102,84 @@ def __init__(self,
                  base_url=None,
                  workers=4,
                  bot=None,
+                 dispatcher=None,
```

Comment:
Please add new kwargs at the end of the parameter list.
I know it should be trivial, but you'd be surprised hom many people still like to use args instead of kwargs